alacritty/src/config/window.rs

```diff
@@ -111,13 +111,15 @@ impl WindowConfig {
         }
     }
 
-    #[cfg(not(any(target_os = "macos", windows)))]
     #[inline]
-    pub fn decorations_theme_variant(&self) -> Option<&str> {
-        self.gtk_theme_variant
-            .as_ref()
-            .or(self.decorations_theme_variant.as_ref())
-            .map(|theme| theme.as_str())
+    pub fn decorations_theme_variant(&self) -> Option<Theme> {
+        self.gtk_theme_variant.as_ref().or(self.decorations_theme_variant.as_ref()).and_then(
+            |theme| match theme.to_lowercase().as_str() {
+                "dark" => Some(Theme::Dark),
+                "light" => Some(Theme::Light),
```

Comment:
I'd assume `Theme` does not implement serialize/deserialize already? Would be easier to just use that in our config.

Reply:
We might want to change it from string to enum?

Reply:
Not sure what you mean.

Reply:
I've updated to use the `enum` instead.